Add Sharded Analytics info (#1306)
TykTechnologies/tyk-docs: developer-support/release-notes/gateway.mdx

## 5.3 Release Notes

> ## 5.3 Release Notes

Reviewer: ```suggestion
```

| [GoLang](https://go.dev/dl/)                               | 1.23       | 1.23       | [Go plugins](/api-management/plugins/golang) must be built using Go 1.23 | 
| [Redis](https://redis.io/download/)  | 6.2.x, 7.x  | 6.2.x, 7.x  | Used by Tyk Dashboard | 
| [OpenAPI Specification](https://spec.openapis.org/oas/v3.0.3) | v3.0.x      | v3.0.x          | Supported by [Tyk OAS](/api-management/gateway-config-tyk-oas#tyk-vendor-extension-reference)|

#### Deprecations

There are no deprecations in this release.

#### Upgrade instructions

If you are upgrading to 5.3.13, please follow the detailed [upgrade instructions](#upgrading-tyk).

#### Downloads

- [Docker image to pull](https://hub.docker.com/r/tykio/tyk-gateway/tags?page=&page_size=&ordering=&name=v5.3.13)
  - ```bash
    docker pull tykio/tyk-gateway:v5.3.13
    ``` 

- Helm charts
  - [tyk-charts v5.0.0](/developer-support/release-notes/helm-chart#500-release-notes)

- [Source code tarball for OSS projects](https://github.com/TykTechnologies/tyk/releases)

#### Changelog
<a id="Changelog-v5.3.13"></a>

##### Fixed

<AccordionGroup>
<Accordion title='Updated Gateway TLS Certificate'>

> Renewed the self-signed certificate shipped in gateway Docker images to prevent expiration (previous certificate expires March 2026). The updated certificate now uses Subject Alternative Names (SAN) instead of the deprecated Common Name (CN) field, ensuring compatibility with modern TLS standards and Go 1.17+. The certificate has a 200-day validity period to meet current security standards. This change affects deployments using `global.tls.gateway=true` in Helm Chart configurations. No immediate action required for existing deployments.

Reviewer: ```suggestion
Renewed the self-signed certificate shipped in gateway Docker images to prevent expiration (previous certificate expires March 2026). The updated certificate now uses Subject Alternative Names (SAN) instead of the deprecated Common Name (CN) field, ensuring compatibility with modern TLS standards and Go 1.17+. The certificate has a 200-day validity period to meet current security standards. 

This change affects deployments using `global.tls.gateway=true` in Helm Chart configurations. No immediate action required for existing deployments.
```

##### 3rd Party Dependencies & Tools

| Third Party Dependency                                     | Tested Versions        | Compatible Versions    | Comments | 
| :---------------------------------------------------------- | :---------------------- | :---------------------- | :-------- | 
| [GoLang](https://go.dev/dl/)                               | 1.23       | 1.23       | [Go plugins](/api-management/plugins/golang) must be built using Go 1.23 | 
| [Redis](https://redis.io/download/)  | 6.2.x, 7.x  | 6.2.x, 7.x  | Used by Tyk Dashboard | 
| [OpenAPI Specification](https://spec.openapis.org/oas/v3.0.3) | v3.0.x      | v3.0.x          | Supported by [Tyk OAS](/api-management/gateway-config-tyk-oas#tyk-vendor-extension-reference)|

#### Deprecations

There are no deprecations in this release.

#### Upgrade instructions

If you are upgrading to 5.3.13, please follow the detailed [upgrade instructions](#upgrading-tyk).

#### Downloads

- [Docker Image to pull](https://hub.docker.com/r/tykio/tyk-dashboard/tags?page=&page_size=&ordering=&name=v5.3.13)
  - ```bash
    docker pull tykio/tyk-dashboard:v5.3.13
    ```

- Helm charts

>   - [tyk-charts v5.0.0](/developer-support/release-notes/helm-chart#500-release-notes)

Reviewer: ```suggestion
- [Docker image to pull](https://hub.docker.com/r/tykio/tyk-gateway/tags?page=&page_size=&ordering=&name=v5.3.13)
  - ```bash
    docker pull tykio/tyk-gateway:v5.3.13
    ``` 

- Helm charts
  - [tyk-charts v5.0.0](/developer-support/release-notes/helm-chart#500-release-notes)

- [Source code tarball for OSS projects](https://github.com/TykTechnologies/tyk/releases)
```

## 5.3 Release Notes

## 5.3 Release Notes

### 5.3.13 Release Notes

#### Release Date xx January 2026

#### Release Highlights

> This is a security patch release to address vulnerabilities in dependent libraries that were present in 5.3.12. For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v5.3.13) below.

Reviewer 2: ```suggestion
This is a version bump to align with Dashboard v5.3.13, no changes have been implemented in this release.
```

#### Breaking Changes

There are no breaking changes in this release.

#### Dependencies

##### Compatibility Matrix For Tyk Components

> | Dashboard Version | Recommended Releases | Backwards Compatibility |

Reviewer 2: ```suggestion
| Gateway Version | Recommended Releases | Backwards Compatibility |
```

| :---- | :---- | :---- |
| 5.3.13 | MDCB v2.8.7     | MDCB v2.8.0 |
|         | Operator v1.2.0  | Operator v0.17 |
|         | Sync v2.1.5    | Sync v2.1.0 |
|         | Helm Chart v5.0  | Helm all versions |

> | | EDP v1.16 | EDP all versions |

Reviewer 2: ```suggestion
```
Gateway does not have a compatibility link with EDP -> that consumes Dashboard, not Gateway.

| | Pump v1.13.2| Pump all versions |
| | TIB (if using standalone) v1.7.0 | TIB all versions |

##### 3rd Party Dependencies & Tools

| Third Party Dependency                                     | Tested Versions        | Compatible Versions    | Comments | 
| :---------------------------------------------------------- | :---------------------- | :---------------------- | :-------- | 

> | [GoLang](https://go.dev/dl/)                               | 1.24       | 1.24       | [Go plugins](/api-management/plugins/golang) must be built using Go 1.24 | 

Reviewer 2: ```suggestion
| [GoLang](https://go.dev/dl/)                               | 1.23       | 1.23       | [Go plugins](/api-management/plugins/golang) must be built using Go 1.23 | 
```

| [Redis](https://redis.io/download/)  | 6.2.x, 7.x  | 6.2.x, 7.x  | Used by Tyk Dashboard | 

> | [MongoDB](https://www.mongodb.com/try/download/community)  | 5.0.x, 6.0.x, 7.0.x  | 5.0.x, 6.0.x, 7.0.x  | Used by Tyk Dashboard | 

Reviewer 2: ```suggestion
```

Gateway does not use Mongo

> | [PostgreSQL](https://www.postgresql.org/download/)         | 13.x - 17.x        | 13.x - 17.x            | Used by Tyk Dashboard | 

Reviewer 2: ```suggestion
```

Gateway does not use SQL

| [OpenAPI Specification](https://spec.openapis.org/oas/v3.0.3) | v3.0.x      | v3.0.x          | Supported by [Tyk OAS](/api-management/gateway-config-tyk-oas#tyk-vendor-extension-reference)|

#### Deprecations

There are no deprecations in this release.

#### Upgrade instructions

If you are upgrading to 5.3.13, please follow the detailed [upgrade instructions](#upgrading-tyk).

#### Downloads

- [Docker Image to pull](https://hub.docker.com/r/tykio/tyk-dashboard/tags?page=&page_size=&ordering=&name=v5.3.13)
  - ```bash
    docker pull tykio/tyk-dashboard:v5.3.13
    ```

- Helm charts
  - [tyk-charts v5.0.0](/developer-support/release-notes/helm-chart#500-release-notes)

#### Changelog
<a id="Changelog-v5.3.13"></a>

> ##### Security Fixes

Reviewer 2: ```suggestion
No changes in this release.
```

> <AccordionGroup>

Reviewer 2: ```suggestion
```

Delete this and the rest of the change log
